mappings/1.6.2-091847#13w36a-09051446.tinydiff

```diff
@@ -219,7 +219,7 @@ c	net/minecraft/unmapped/C_0539808
 		p	6			block
 		p	7			power
 	m	()Lnet/minecraft/unmapped/C_7541710;	m_5594379		m_5594379
-c	net/minecraft/unmapped/C_0543602	net/minecraft/world/biome/IceBiome	
+c	net/minecraft/unmapped/C_0543602	net/minecraft/world/biome/ExtremeHillsBiome	
```

Comment:
This is a problematic change, 'cause it indicates there is likely a bad match somewhere between 1.0 and 1.6 - I won't let it stand in the way of merging this PR, but that mapping will need to be fixed properly at some point.

Reply:
after a bit of research I found that the issue is between 12w19a and 12w21a

Reply:
> I won't let it stand in the way of merging this PR

oops, spoken too soon - that name leads to mapping conflicts, which is why the workflow failed - I'm afraid it'll have to be fixed before I merge this

Reply:
What's the way to fix it? Do I need to do something on this PR right now? Also:
> indicates there is likely a bad match somewhere between 1.0 and 1.6 

Since I mapped stuff like the `ScreenshotUtils` class (and some others) that already existed when I looked at 1.4, does that also indicate there's a bad match (but in the other way around, since it's missing)?

Reply:
> What's the way to fix it? Do I need to do something on this PR right now?

The server-client split makes this a little trickier, but I think the easiest way to fix it will be to revert this change, then open Enigma for 12w19a to update the mapping, and save the changes with the `propagateMappingsDown` task.

> Also:
>
> > indicates there is likely a bad match somewhere between 1.0 and 1.6
>
> Since I mapped stuff like the ScreenshotUtils class (and some others) that already existed when I looked at 1.4, does that also indicate there's a bad match (but in the other way around, since it's missing)?

yeah probably, the vast version ranges really expose how fragile and imperfect intermediary is with issues like these

Reply:
Well I had a lot of trouble with this (I assume that's kinda normal? 🙂) But I think I got it to work now. The `python feather.py clean build checkMappings mapNamedJar` command works locally for b1.7, 1.0.0, 1.6.4, and 1.12.2.

I also noticed that some other biomes have similar issues (Plains is named `StoneBeachBiome`, and ocean and river biome class names are swapped) but I do not feel confident in fixing that at this time. Maybe you or someone else who has more experience with this can do it instead.

Reply:
Excellent, the workflow now succeeds as well! Thank you for looking into it a more deeply. 

> Well I had a lot of trouble with this (I assume that's kinda normal? 🙂)

Yeah, the build script is giving quite some people a lot of trouble. I can give one tip on an issue I've encountered before, which is to make sure to run the `separateMappings` task after the propagation task, if you intend to keep working on the mappings in Enigma. If you do not, then you effectively erase all of the mappings, since the propagation tasks delete the working files afterwards.